data/json/items/melee/bludgeons.json

```
@@ -453,6 +453,26 @@
     "melee_damage": { "bash": 22, "cut": 8 },
     "weapon_category": [ "MACES" ]
   },
+  {
+    "type": "GENERIC",
+    "id": "nutboltbat",
+    "looks_like": "nailbat",
+    "name": { "str": "bolt studded bat" },
+    "description": "A baseball bat with bolts drilled into it, concentrating the force of an impact onto a smaller point like a real mace.",
+    "weight": "1181 g",
+    "price_postapoc": "12 USD 50 cent",
+    "color": "brown",
+    "symbol": "/",
+    "material": [ "wood", "steel" ],
+    "techniques": [ "WBLOCK_1", "BRUTAL" ],
+    "flags": [ "DURABLE_MELEE", "NONCONDUCTIVE", "BELT_CLIP" ],
```

```suggestion
    "flags": [  "NONCONDUCTIVE", "BELT_CLIP" ],
```
It's not fragile but it's also not a durable melee item.

---

Agreed. I should apply this to the nail bat too.